docs/modules/migrate/pages/which-tool-to-use.adoc

```diff
@@ -0,0 +1,29 @@
+= Which Tool Should I Use?
+:description: This flow chart will help you determine which upgrade or migration tool you should use for your Hazelcast cluster.
+
+{description} 
+
+Hazelcast provides several different options for upgrading your Hazelcast cluster and/or migrating your data from one cluster to another. 
+
+* The Data Migration Tool (DMT): Migrate data from your Open Source or Enterprise clusters running Hazelcast 4.0 or later, to either an Enterprise cluster or a cluster on Hazelcast Viridian. xref:data-migration-tool.adoc[]
+* Data Synchronization using Active/Active WAN Replication: Used to migrate data and active clients from one Enterprise cluster to another Enterprise cluster. xref:wan:wan.adoc[]
+* 3.12 Migration jar: A specialized tool for migrating data only from a 3.12 cluster to the latest version of Hazelcast. xref:migration-tool.adoc[]
```

Comment:
this link is not looking good 
<img width="1693" alt="image" src="https://github.com/hazelcast/hz-docs/assets/32355782/bb4380c2-0339-4d72-9976-4f45c858b003">

Reply:
I suppose this is because the page is not published yet? Anyway, you guys know this stuff better